configs.json

```diff
@@ -1,10 +1,16 @@
 {
   "vehicle_type": "Quadcopter",
-  
   "vehicle_id": 1,
   "vehicle_simulated": true,
   "coms_simulated": true,
   "comm_sim_file": "comm_sim_example.json",
+  "dest": {
+    "lat": 35.3284237,
+    "long": -120.7530046,
+    "lat_long_margin_of_error": 1,

```

Comment:
Can we move these out of the dest object? they are relevant elsewhere in the code.